src/tags/everyone.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * @Author: stupid cat
+ * @Date: 2017-05-07 18:38:55
+ * @Last Modified by: stupid cat
+ * @Last Modified time: 2017-10-19 18:12:29
+ *
+ * This project uses the AGPLv3 license. Please read the license file before using/adapting any of the code.
+ */
```

Comment:
In this case, who would be the author? Is it the author of the file, or just Stupid Cat?

Reply:
> In this case, who would be the author? Is it the author of the file, or just Stupid Cat?

To add on this, I'm not familiar at all with jsdoc. Is there any good source that explains the importance of it?

Reply:
> To add on this, I'm not familiar at all with jsdoc. Is there any good source that explains the importance of it?

@RagingLink It is difficult to give you a proper answer because the JSDoc tags used by @Ratismal are not common. This is more like a "file disclaimer" instead of a real "documentation".
Howerer, here's [a resource about JSDoc](https://jsdoc.app) tags used on classes, methods, class variables, etc.